wtxmgr/query_test.go

```diff
@@ -525,31 +534,23 @@ func TestStoreQueries(t *testing.T) {
 	newState.txDetails[recA.Hash] = append(newState.txDetails[recA.Hash], newState.blocks[1][0])
 	lastState = newState
 	tests = append(tests, queryTest{
-		desc:    "insert duplicate tx A",
-		updates: func(ns walletdb.ReadWriteBucket) { insertTx(ns, recA, &b100); insertTx(ns, recA, nil) },
-		state:   newState,
-	})
-	newState = lastState.deepCopy()
-	newState.blocks = [][]TxDetails{
-		newState.blocks[0],
-		{newState.blocks[1][0]},
-		{newState.blocks[1][1]},
-	}
-	newState.blocks[1][0].Block = b101
-	newState.txDetails[recA.Hash][1].Block = b101
-	lastState = newState
-	tests = append(tests, queryTest{
-		desc:    "mine duplicate tx A",
```

Comment:
what happened to this test?

Reply:
Previously, the store would allow storing the same hash in both unconfirmed and confirmed buckets at the same time. We no longer allow this so the test had to be removed, otherwise it'd cause a failure.